packages/agent/src/sync-engine-level.ts

```diff
@@ -355,13 +355,14 @@ export class SyncEngineLevel implements SyncEngine {
     syncDirection: SyncDirection,
     cursor?: PaginationCursor
   }) {
-    let eventsReply = {} as EventsGetReply;
+    let eventsReply = {} as EventsQueryReply;
 
     if (syncDirection === 'pull') {
       // When sync is a pull, get the event log from the remote DWN.
       const eventsGetMessage = await this.agent.dwn.createMessage({
         author        : did,
-        messageType   : DwnInterface.EventsGet,
+        messageType   : DwnInterface.EventsQuery,
+        // TODO: Liran help!
         messageParams : { cursor }
```

Comment:
Hey @LiranCohen, `messageParams` current broke, need your help instead of me wasting time fixing, I know you know exactly what needs to be done.

You can directly checkout and merge in fixes in this branch!

Reply:
Thanks @LiranCohen, you are the best.